src/NuGetGallery/Controllers/AccountsController.cs

```diff
@@ -52,6 +52,8 @@ public class ViewMessages
 
         public IDeleteAccountService DeleteAccountService { get; }
 
+        private readonly DeleteAccountListPackageItemViewModelHelper _deleteAccountListPackageItemViewModelHelper;
```

Comment:
What're your thoughts on renaming `Helper` to `Builder`?

```suggestion
        private readonly DeleteAccountListPackageItemViewModelBuilder _deleteAccountListPackageItemViewModelBuilder;
```

Reply:
Have you considered `Factory` as well?

Reply:
I think [Builder](https://en.wikipedia.org/wiki/Builder_pattern) is an established pattern when one create a stateful builder object, throws bunch of items at it, then asks it to produce an instance of some target object. These guys are essentially stateless.

Reply:
Sold on "Factory"